src/ListArray.java

```diff
@@ -0,0 +1,158 @@
+public class ListArray {
+
+    private int length;
+    private int[] content;
+
+    public ListArray(int length, int[] contents){
+        this.length = length;
+        this.content = new int[length];
```

Comment:
You do not need this `for` loop; your `ArrayList` should initialize without values.